src/assembleLookupResults.js

```diff
@@ -39,7 +39,11 @@ const getResultsForThisEntity = (entity, alerts, indicators, events, options) =>
 const createSummaryTags = ({ alerts, indicators, events }, options) =>
   []
     .concat(size(alerts) ? `Alerts: ${size(alerts)}` : [])
-    .concat(size(indicators) ? `Indicators: ${size(indicators)}` : [])
+    .concat(
+      size(indicators) && size(indicators) > 1
+        ? `Indicators: ${size(indicators)}`
+        : `Max Severity: ${indicators[0].maxSeverity}`
```

Comment:
@ntsmith7 Is there a risk here that `indicators` is undefined?  Do we need to guard against that here and make sure we're not attempting to access a property on an undefined value?